aws/cf_template.go

```diff
@@ -300,11 +300,11 @@ func generateTemplate(spec *stackSpec) (string, error) {
 		)
 	}
 
-	if spec.nlbCrossZone && spec.loadbalancerType == LoadBalancerTypeNetwork {
+	if spec.loadbalancerType == LoadBalancerTypeNetwork {
 		lbAttrList = append(lbAttrList,
 			cloudformation.ElasticLoadBalancingV2LoadBalancerLoadBalancerAttribute{
 				Key:   cloudformation.String("load_balancing.cross_zone.enabled"),
-				Value: cloudformation.String("true"),
+				Value: cloudformation.String(fmt.Sprintf("%t", spec.nlbCrossZone)),
```

Comment:
This allows disabling of crosszone for the existing NLBs created with cross-zone enabled because before the change this would just omit the attribute.
Is my understanding correct here?

Reply:
Yes

Reply:
Are you sure you can change this on an existing NLB? I think it was left out because you can't change it after creation.

Reply:
Are you sure?
At least I can do this in the AWS console by clicking the edit button on attributes
![image](https://github.com/zalando-incubator/kube-ingress-aws-controller/assets/50872/81ebee89-9830-40c8-b520-a33f5ac67de5)

Reply:
I tested it in our dev cluster and it works as expected. Maybe in the past it was not possible.

Reply:
Here random NLB from the dev cluster:
![image](https://github.com/zalando-incubator/kube-ingress-aws-controller/assets/50872/ae13ebd0-c8f9-4da9-b7be-d27c9679c08d)